src/EditorFeatures/Core/ExternalAccess/IntelliCode/Api/IIntentSourceProvider.cs

```diff
@@ -28,7 +27,7 @@ internal interface IIntentSourceProvider
 internal readonly struct IntentRequestContext(string intentName, SnapshotSpan currentSnapshotSpan, ImmutableArray<TextChange> textEditsToPrior, TextSpan priorSelection, string? intentData)
```

Comment:
Is this code already dead, so it's just a matter of time of deleting the code from the callers too?

Reply:
yes.